Cargo.lock

```diff
@@ -48,21 +59,29 @@ dependencies = [
 
 [[package]]
 name = "ark-algebra-test-templates"
-version = "0.3.0"
+version = "0.4.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "3eef0b339ebc113d9bd3fb7cd666baf2cfe4e1024e0fac23e072d46598bbd0cd"
+checksum = "400bd3a79c741b1832f1416d4373ae077ef82ca14a8b4cee1248a2f11c8b9172"
 dependencies = [
  "ark-ec",
  "ark-ff",
  "ark-serialize",
  "ark-std",
+ "hex",
+ "num-bigint",
+ "num-integer",
+ "num-traits",
+ "serde",
+ "serde_derive",
+ "serde_json",
+ "sha2",
 ]
 
 [[package]]
 name = "ark-bn254"
```

Comment:
Next step is to get rid of this. It is not in arkworks. Opening follow-up issue.